ts/utils/idps.ts

```diff
@@ -1,79 +1,82 @@
+import { ImageSourcePropType } from "react-native";
 import { SpidIdp } from "../../definitions/content/SpidIdp";
 // since this is a test SPID idp, we set isTestIdp flag to avoid rendering.
 // It is used has a placeholder to handle taps count on it and open when
 // taps count threadshold is reached (see https://www.pivotaltracker.com/story/show/172082895)
 
-export const testIdp: SpidIdp = {
+export type LocalIdpsFallback = SpidIdp & { localLogo?: ImageSourcePropType };
```

Comment:
There is a casa in which `localLogo` will be undefined? Can be safest to set is as mandatory?

Reply:
yep, in case of testIDP we don't have the logo since we don't show any item